### ISSUE TITLE
Microsoft PowerPlatform CLI - "Power Platform Import Solution"

### ISSUE THREAD

**Issue by demasj** [microsoft/powerplatform-build-tools#569]
Our team is suddenly receiving error messages when they initiate a release to the different environments. It fails at the 'Power Platform Import Solution' step with the following error message. Is this already known, and is there a solution for this?

2023-10-04T11:15:11.0687613Z ##[error]error: 1
2023-10-04T11:15:11.0699230Z ##[error]failed: Connected to... ***
Connected as ***

Solution Importing...
Microsoft PowerPlatform CLI
Version: 1.27.6+g9f62afb
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

Error: An unexpected error occurred.
    An unexpected error occurred.

**Comment by snizar007** [comment]
Can you please share the pac-log.txt?
log file path can be found using "pac telemetry status".
Will take a look. Thanks for reporting.

**Comment by petrochuk** [comment]
It also helps to run your pipeline with "Enable Diagnostics" which will give you more details about the error

**Comment by demasj** [comment]
> It also helps to run your pipeline with "Enable Diagnostics" which will give you more details about the error

I am using the classic pipeline, and I believe there is no option for 'Enable Diagnostics' in it. Currently, I am in the process of converting the classic pipeline to a YAML pipeline. The first stage, 'UAT,' is working well for me now

**Comment by demasj** [comment]
"I have converted my classic pipeline to YAML, and now I was able to enable the 'Enable Diagnostics' option. Please see the attached LOG file for reference.
[Log.txt](https://github.com/microsoft/powerplatform-build-tools/files/12832011/Log.txt)

**Comment by petrochuk** [comment]
You can see the error:
```
Request failed with: Forbidden and error: {"error":{"code":"ConnectionAuthorizationFailed",
"message":"The caller with object id '****************' does not have the minimum required permission 
to perform the requested operation on connection '*********' under API 'shared_sql'."}} and request url 
```

**Comment by demasj** [comment]
Yes, I noticed that as well. The Service Principal Name (SPN) I am using has both the 'System Administrator' and 'System Customizer' roles in the environment. Although I suspect that the latter role may not even be necessary.

**Comment by demasj** [comment]
After I shared the 'Connection' with the Service Principal Name (SPN) that the pipeline is using, it started working.
